SpanishFilter/sections/specific.txt

```diff
@@ -1543,6 +1537,7 @@ universoformulas.com##.post-inner > div[style="margin: 0 0 25px 0"] > span[style
 torrentbrazil.org##a[href^="https://goo.gl/"] > img
 jn.pt##div[class^="t-pub-box-"]
 televalltv.net###floatLayer6
+cinemaniahdd.com##a[href^="//mellowads.com"]
 tufutbolpro.com##a[href^="http://bit.ly/"]
```

Comment:
@SKEIDs 
It seems you forgot to delete this rule.

Reply:
I mistook to solve conflicts